playbooks/collect-logs.yml

```diff
@@ -114,6 +118,36 @@
         segfault_problems_found: "{{ result.stdout_lines }}"
       ignore_errors: yes
 
+    - name: Collect logs from nova
+      sudo: yes
+      shell: cp -Rf /var/log/nova/* /var/log/extra/openstack
+      ignore_errors: yes
+
+    - name: Collect logs from glance
+      sudo: yes
+      shell: cp -Rf /var/log/glance/* /var/log/extra/openstack
+      ignore_errors: yes
+
+    - name: Collect logs from cinder
+      sudo: yes
+      shell: cp -Rf /var/log/cinder/* /var/log/extra/openstack
+      ignore_errors: yes
+
+    - name: Collect logs from keystone
+      sudo: yes
+      shell: cp -Rf /var/log/keystone/* /var/log/extra/openstack
+      ignore_errors: yes
+
+    - name: Collect logs from neutron
+      sudo: yes
+      shell: cp -Rf /var/log/neutron/* /var/log/extra/openstack
+      ignore_errors: yes
+
+    - name: Collect logs from horizon
+      sudo: yes
+      shell: cp -Rf /var/log/httpd/horizon_error.log /var/log/extra/openstack
+      ignore_errors: yes
+
```

Comment:
I though about that, but, what if in the middle, it doesn't find for example the glance files, the with_items will continue with the others or it will fail and move to next task?